app/webpacker/components/BaseCalendarManager.tsx

```diff
@@ -94,8 +94,8 @@ export abstract class BaseCalendarManager<TProps extends BaseCalendarManagerProp
     const { classes } = this.props;
     const event = eventInfo.event;
     const extendedProps = event.extendedProps;
-    const summary = [event.title, extendedProps.spot?.name, extendedProps.field?.name, " Participants ", extendedProps.amountParticipants].filter(Boolean).join(' - ');
-    
+    const summary = [event.title, extendedProps.spot?.name, extendedProps.field?.name, " 👥 " + extendedProps.amountParticipants, extendedProps.languages?.join(", ")].filter(Boolean).join(' - ');
```

Comment:
nice 👍 make sure that this is really the right number and not the expected number or so